sap/xssec/security_context.py

```diff
@@ -6,16 +6,12 @@
 import logging
 import requests
 import deprecation
+import urllib3
 
 from sap.xssec import constants
 from sap.xssec.jwt_validation_facade import JwtValidationFacade, DecodeError
 from sap.xssec.key_cache import KeyCache
 
-try:
```

Comment:
Does this also work for Python 2? I think the point of this import was to have code working for both Python 2 and 3. I believe `cloud-pysec` claims official support for Python 2: https://github.com/SAP/cloud-pysec#requirements

Reply:
my bad, I'll check it tonight. The unit-tests run also on python 2, so I guess it works, but I'll double check

Reply:
urllib3 supports Python >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4